crypto/x509/x509cset.c

```diff
@@ -134,7 +134,8 @@ int X509_CRL_sort(X509_CRL *c)
 
 void X509_CRL_up_ref(X509_CRL *crl)
```

Comment:
Most of <Type>_up_ref() methods are returning an int, except a few like this one.
Is it a matter for a separated PR ?